skyfield/documentation/positions.rst

```diff
@@ -39,7 +39,8 @@ together with all of the attributes and methods that they support:
 
     Apparent position only
      │
-     └── `altaz(…) <api.html#Position.altaz>`_            →   alt, az, distance
+     └── `altaz(…) <api.html#Position.altaz>`_                 →   alt, az, distance
+     └── `over_location(…) <api.html#Apparent.over_location>`_ →   topos
```

Comment:
You are entirely correct that names are important, and I am indeed wanting people new to astronomy to be a primary audience for Skyfield. But, sometimes, if we can find a more “official” name for a concept, then the Skyfield user — while thinking they are just writing a quick program — is also learning official terminology that will help them out when the look at other programs and web sites. The name does not stand in the way of the pull request being merged, so we can keep it this way until I have time at PyCon to look into what other libraries call this idea. Thanks!